application/composer.json

```diff
@@ -15,11 +15,11 @@
     "silinternational/yii2-email-log-target": "^1.0",
     "theiconic/php-ga-measurement-protocol": "^2.0",
     "yiisoft/yii2-swiftmailer": "^2.0",
-    "ramsey/uuid": "^3.6"
+    "ramsey/uuid": "^3.6",
+    "guzzlehttp/guzzle": "^6.2"
   },
   "require-dev": {
     "behat/behat": "^3.3",
-    "guzzlehttp/guzzle": "^6.2",
     "webmozart/assert": "^1.2",
     "friendsofphp/php-cs-fixer": "^2.14"
   },
```

Comment:
When I install the composer dependencies, it says that the lock file is out of sync with the composer.json file, so you may need to do another `make depsupdate`.

Reply:
okay, I did that, will include in a batch of updates shortly.